project_multifamily_beta/housing_characteristics/ASHRAE IECC Climate Zone 2004.tsv

```diff
@@ -0,0 +1,3 @@
+Option=1A	Option=2A	Option=2B	Option=3A	Option=3B	Option=3C	Option=4A	Option=4B	Option=4C	Option=5A	Option=5B	Option=6A	Option=6B	Option=7A	Option=7B	sample_count	sample_weight
+0.013303	0.111950	0.019338	0.131728	0.097040	0.025144	0.218931	0.007254	0.030202	0.231173	0.038954	0.059321	0.008457	0.006529	0.000676	72538.000000	116975856.000000
+# Created by: sources\spatial\tsv_maker.py using unit counts from the American Community Survey 5-yr 2016; spatial data from U.S. Census 2010; climate zone data from ASHRAE 169 2006, IECC 2004, and M.C. Baechler 2015; ISO and RTO regions from EIA Form 861.
```

Comment:
"spatial data from U.S. Census 2010" What does this mean?
"climate zone data from ASHRAE 169 2006, IECC 2004" I have questions about these years. If IECC adopts the ASHRAE zone, why is the year 2004 before 2006?

Reply:
- "Spatial data" refers to all the geospatial descriptions: Census tracts, County, PUMA, States, Census Divisions, and Census Regions.
- The climate zone descriptions are taken from the following report. VOLUME 7.3 Guide to Determining Climate Regions by County by PNNL in 2015 (link below). In this document, PNNL states "The PNNL-developed map was adopted by the IECC and was first included in the IECC in the 2004 Supplement to the IECC. It first appeared in ASHRAE 90.1 in the 2004 edition." So both IECC and ASHRAE (through 90.1) first adopted the map in 2004. In the created by line, I am referencing the climate standard ASHRAE 169, which was updated in 2006.

https://www.energy.gov/sites/prod/files/2015/10/f27/ba_climate_region_guide_7.3.pdf